source/user-manual/user-administration/password-management.rst

```diff
@@ -198,7 +198,7 @@ Follow the instructions below to change the passwords for all the Wazuh indexer
       INFO: The password for user logstash is nQg1Qw0nIQFZXUJc8r8+zHVrkelch33h
       INFO: The password for user readall is s0iWAei?RXObSDdibBfzSgXdhZCD9kH4
       INFO: The password for user snapshotrestore is Mb2EHw8SIc1d.oz.nM?dHiPBGk7s?UZB
-      WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, and Filebeat nodes if necessary, and restart the services.
+      WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, keystore tool, and Filebeat nodes if necessary, and restart the services.
```

Comment:
Ditto

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/166a6616aa43bcfd53cffd26dd40a3764a21f901) commit